### PR TITLE
added changes for the affordability widget.

### PR DESCRIPTION
**Pull request by yashgit891**

### CODE REVIEW

File: includes/razorpay-affordability-widget.php [hunk 1]
```diff
@@ -0,0 +1,529 @@
+<?php
+
+require_once __DIR__.'/utils.php';
```
Comment (abdulwahidsharief) [inline includes/razorpay-affordability-widget.php:3]: @yashgit891 leave a blank line after imports

File: includes/razorpay-affordability-widget.php [hunk 2]
```diff
@@ -0,0 +1,529 @@
...
+function addAffordabilityWidgetHTML()
+{
+    $current_user = wp_get_current_user();
+    if ((isAffordabilityWidgetTestModeEnabled() == false and empty(get_option('rzp_afd_enable')) == false and  get_option('rzp_afd_enable') == 'yes')
```
Comment (abdulwahidsharief) [inline includes/razorpay-affordability-widget.php:7]: use === and every new condition should start with new line

File: includes/razorpay-affordability-widget.php [hunk 3]
```diff
@@ -0,0 +1,529 @@
...
+            const amount = parseFloat("'.getPrice().'s") * 100;
+            window.onload = function() {
+                const widgetConfig = {
+                        "key": key,
+                        "amount": amount,
+                        "theme": {
+                        "color": "'.getThemeColor().'"
+                        },
+                        "features": {
+                        "offers": {
```
Comment (abdulwahidsharief) [inline includes/razorpay-affordability-widget.php:26]: check indentation

File: includes/razorpay-affordability-widget.php [hunk 4]
```diff
@@ -0,0 +1,529 @@
...
+
+function getKeyId()
+{
+    return get_option('woocommerce_razorpay_settings')['key_id'];
+}
+
+function getPrice()
+{
+    global $product;
+    if (!$product->is_type('variable'))
```
Comment (abdulwahidsharief) [inline includes/razorpay-affordability-widget.php:75]: use boolean comparison

File: includes/razorpay-affordability-widget.php [hunk 5]
```diff
@@ -0,0 +1,529 @@
...
+        else
+        {
+            $price = $product->get_regular_price();
+        }
+    }
+    else
+    {
+        $price = $product->get_price();
+    }
+    return $price;
```
Comment (abdulwahidsharief) [inline includes/razorpay-affordability-widget.php:89]: leave blank line before return

**Review by abdulwahidsharief — COMMENTED** [review]
please follow code formatting guidelines  https://docs.google.com/document/d/1GZTKZVngt08YzbqvFt15KIHxLXCOo48ly1Q0XSO8rUY/edit?pli=1#heading=h.7hl6krfv2iid

File: woo-razorpay.php [hunk 6]
```diff
@@ -2172,6 +2199,18 @@ function razorpay_woo_plugin_links($links)
     }
 
     add_filter('plugin_action_links_' . plugin_basename(__FILE__), 'razorpay_woo_plugin_links');
+
+    try
+    {
+        $api = new Api(get_option('woocommerce_razorpay_settings')['key_id'], get_option('woocommerce_razorpay_settings')['key_secret']);
+        $validateKeySecret = $api->request->request('GET', 'accounts/me/features');
```
Comment (sonyy-m) [inline woo-razorpay.php:2206]: remove these unused lines of code

File: includes/razorpay-affordability-widget.php [hunk 7]
```diff
@@ -0,0 +1,497 @@
...
+    ];
+
+    $customisationValue = get_option($customisation);
+    if (empty($customisationValue) === true)
+    {
+        $customisationValue = $defaultCustomisationValues[$customisation];
+    }
+    
+    return $customisationValue;
+}
```
Comment (sonyy-m) [inline includes/razorpay-affordability-widget.php:497]: Add EOL char

**Review by sonyy-m — APPROVED** [review]
LGTM

**Pull request merged**